src/app/reporting/ReportScheduler.h

```diff
@@ -56,6 +56,17 @@ class ReportScheduler : public ReadHandler::Observer
     class ReadHandlerNode : public IntrusiveListNodeBase<>
     {
     public:
+#ifdef CONFIG_BUILD_FOR_HOST_UNIT_TEST
+        /// Test flags to allow TestReadInteraction to simulate expiration of the minimal and maximal intervals without
```

Comment:
Couldn't it just poke the timestamps to be <= now as needed?   Or in fact change the concept of "now"....  Or inject a timer delegate that advances its time and fires its timers faster, or whatever.

I'm not going to block on this, but a followup to remove this, please.  It shouldn't be here.

Reply:
Issue opened here: https://github.com/project-chip/connectedhomeip/issues/28129